### PR TITLE
OCPBUGS#3374: Added note for globalnet

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s):4.11+

Issue: https://issues.redhat.com/browse/OCPBUGS-3374

Link to docs preview:

- Alibaba: https://53459--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_alibaba/installing-alibaba-network-customizations.html#installation-configuration-parameters-network_installing-alibaba-network-customizations
- AWS: https://53459--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-network-customizations.html#installation-configuration-parameters-network_installing-aws-network-customizations
- Azure: https://53459--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-network-customizations.html#installation-configuration-parameters-network_installing-azure-network-customizations
- Azure Stack Hub: https://53459--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure_stack_hub/installing-azure-stack-hub-network-customizations.html#installation-configuration-parameters-network_installing-azure-stack-hub-network-customizations
- GCP: https://53459--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-gcp-network-customizations.html#installation-configuration-parameters-network_installing-gcp-network-customizations
- IBM Cloud: https://53459--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_ibm_cloud_public/installing-ibm-cloud-network-customizations.html#installation-configuration-parameters-network_installing-ibm-cloud-network-customizations
- Bare metal: https://53459--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal-network-customizations.html#installation-configuration-parameters-network_installing-bare-metal-network-customizations
- OpenStack: https://53459--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-custom.html#installation-configuration-parameters-network_installing-openstack-installer-custom
- RHV: https://53459--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_rhv/installing-rhv-customizations.html#installation-configuration-parameters-network_installing-rhv-customizations
- vSphere: https://53459--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.html#installation-configuration-parameters-network_installing-vsphere-installer-provisioned-network-customizations
- VMC: https://53459--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_vmc/installing-vmc-network-customizations.html#installation-configuration-parameters-network_installing-vmc-network-customizations

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by olivelakra** [comment]
@snarayan-redhat updates are LGTM

File: modules/installation-configuration-parameters.adoc [hunk 1]
```diff
@@ -338,6 +338,11 @@ networking:
   - fd00:172:16::/112
 ----
 
+[NOTE]
+====
+Globalnet is not supported with Red Hat OpenShift Data Foundation disaster recovery solutions. Ensure that you use a non-overlapping range of private IP addresses for the cluster and service networks in each cluster, for regional disaster recovery scenarios.
```
Comment (jeana-redhat) [inline modules/installation-configuration-parameters.adoc:343]: - Is "Globalnet" in this note the company "[GlobalNet](https://www.gblnet.net/en)"? if so, capitalization should be updated.
- "Red Hat OpenShift Data Foundation" has an [attribute](https://github.com/openshift/openshift-docs/blob/main/_attributes/common-attributes.adoc), `rh-storage-first`
- Also, a couple of small style tweaks:

```suggestion
Globalnet is not supported with {rh-storage-first} disaster recovery solutions. For regional disaster recovery scenarios, ensure that you use a nonoverlapping range of private IP addresses for the cluster and service networks in each cluster.
```

**Review by jeana-redhat — COMMENTED** [review]
A couple fixes here

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/installation-configuration-parameters.adoc (hunk 1 above)
Comment (snarayan-redhat) [reply modules/installation-configuration-parameters.adoc:343]: - This Globalnet is not the company GlobalNet.
- Addressed
- Addressed

**Pull request merged**